Packs/Dropbox/Playbooks/DropBox_-_Massive_scale_operations_on_files.yml

```diff
@@ -0,0 +1,964 @@
+    isautoswitchedtoquietmode: false
+  "55":
+    id: "55"
+    taskid: 64376ca5-5a8a-43d2-881b-e22f719facf1
+    type: playbook
+    task:
+      id: 64376ca5-5a8a-43d2-881b-e22f719facf1
+      version: -1
+      name: Enrichment for Verdict
+      description: This playbook checks prior alert closing reasons and performs enrichment and prevalence checks on different IOC types. It then  returns the information needed to establish the alert's verdict.
+      playbookName: Enrichment for Verdict
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "28"
+    scriptarguments:
+      CloseReason:
+        simple: Resolved - False Positive,Resolved - Duplicate Incident,Resolved - Known Issue
+      Domain:
+        complex:
+          root: alert
+          accessor: domainname
+      FileSHA256:
+        complex:
+          root: alert.initiatorsha256
+          accessor: '[0]'
+      IP:
+        complex:
+          root: alert.remoteip
+          accessor: '[0]'
+          transformers:
+          - operator: uniq
+      InternalRange:
+        complex:
+          root: inputs.InternalRange
+      ProcessName:
+        complex:
+          root: alert
+          accessor: initiatedby
+          transformers:
+          - operator: split
+            args:
+              delimiter:
+                value:
+                  simple: ', '
+      URL:
+        complex:
+          root: alert
+          accessor: url
+      User:
+        complex:
+          root: alert
+          accessor: username
+          transformers:
+          - operator: uniq
+      query:
+        simple: remoteip:${alert.remoteip.[0]} and alertsource:${alert.sourceBrand} and alertname:"${alert.name}"
+      threshold:
+        simple: "5"
+    separatecontext: false
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 370,
+          "y": 60
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "28_20_Malicious IP or Abnormal File": 0.38,
+      "33_12_Yes": 0.19,
+      "33_20_#default#": 0.16,
+      "36_30_#default#": 0.52,
+      "36_40_Block Both IP Address and User Account": 0.81,
+      "36_41_Block IP Address Only": 0.81,
+      "53_29_Found User Email Addrerss": 0.78,
+      "53_30_#default#": 0.12,
+      "53_51_Found Manager Email Address": 0.76,
+      "54_12_yes": 0.42,
+      "54_20_#default#": 0.1
+    },
+    "paper": {
+      "dimensions": {
+        "height": 2105,
+        "width": 2340,
+        "x": -710,
+        "y": -240
+      }
+    }
+  }
+inputs:
+- key: InternalRange
+  value: {}
+  required: false
+  description: List of Internal IP ranges
+  playbookInputQuery:
+outputs: []
```

Comment:
we want to the playbook available for XSIAM only.
Please add the following in the playbook yml file.
`marketplaces: ["marketplacev2"]`